### PR TITLE
prepare for bench additions

### PR DESCRIPTION
**Pull request by TorreZuk**
* prepares for adding bench code by standardizing

### CODE REVIEW

File: clients/include/testing_geam.hpp [hunk 1]
```diff
@@ -21,7 +21,7 @@ using namespace std;
 /* ============================================================================================ */
```
Comment (daineAMD) [inline clients/include/testing_geam.hpp:21]: no `#include "testing_common.hpp"` here?

File: clients/include/testing_set_get_atomics_mode.hpp [hunk 2]
```diff
@@ -19,7 +19,7 @@ using namespace std;
```
Comment (daineAMD) [inline clients/include/testing_set_get_atomics_mode.hpp:19]: Same here and in set_get_vector and set_get_vector_async

**Review by daineAMD — APPROVED** [review]
LGTM

File: clients/include/testing_set_get_atomics_mode.hpp (hunk 2 above)
Comment (TorreZuk) [reply clients/include/testing_set_get_atomics_mode.hpp:19]: Well if they had few includes I didn't update.  I guess I can change them too, unless they won't get benched like atomics, unless maybe we should bench atomics.

File: clients/common/hipblas_arguments.cpp [hunk 3]
```diff
@@ -16,6 +16,7 @@ std::ostream& operator<<(std::ostream& os, std::pair<T1, T2> p)
 {
     os << p.first << ": ";
     os << p.second;
+    return os;
```
Comment (cgmb) [inline clients/common/hipblas_arguments.cpp:19]: You can turn this sort of omission into a compiler error with the flag `-Werror=return-type`. It has basically no false-positives, but I suppose enabling *any* warnings as errors at all is probably a bad idea for a default in an OSS library. We could probably add an option that the CI opts into, though.

Comment (TorreZuk) [reply clients/common/hipblas_arguments.cpp:19]: Yes we don't do that for many warnings, usually fixing any warnings is the best policy.   Leave it to @daineAMD  to decide if he wants to adjust hipBLAS I am fine either way.

**Pull request merged**